scout/adapter/mongo/hgnc.py

```diff
@@ -371,6 +371,17 @@ def genes_by_alias(self, build="37", genes=None):
 
         return alias_genes
 
+    def ensembl_to_hgnc_mapping(self):
+        """Return a dictionary with Ensembl ids as keys and hgnc_ids as values
+
+        Returns:
+            mapping(dict): {"ENSG00000121410":"A1BG", ...}
+        """
+        pipeline = [{"$group": {"_id": {"ensembl_id": "$ensembl_id", "hgnc_id": "$hgnc_id"}}}]
+        result = self.hgnc_collection.aggregate(pipeline)
```

Comment:
This query is amazingly fast I think. Tested loading all PanelApp panels

Reply:
Also not using indexes! Well implemented functionality. It is a decently small collection after all, so long as we don't get tons of competing builds.